tutorials/data_generation_ppi.ipynb

```diff
@@ -102,7 +102,9 @@
     "data_path = os.path.join(\"data_raw\", \"ppi\")\n",
     "processed_data_path = os.path.join(\"data_processed\", \"ppi\")\n",
     "os.makedirs(os.path.join(processed_data_path, \"residue\"))\n",
-    "os.makedirs(os.path.join(processed_data_path, \"atomic\"))"
+    "os.makedirs(os.path.join(processed_data_path, \"atomic\"))\n",
```

Comment:
By setting `exist_ok` to `True` (also needs to be done above and in other notebook), the notebook can be run a second time without having to remove the folder created during previous run.

```suggestion
    "os.makedirs(os.path.join(processed_data_path, \"atomic\", exist_ok=True))\n",
```

Reply:
But if you've already processed data, it will keep adding hdf5 files to the same folder, even if during the second run you may have edited the settings of the processing. I think that the user can edit by himself/herself the code in such cases if he/she means to do that. We're providing the notebooks to show them a working pipeline, we don't have to consider all the possibilities in this regard.